packages/terra-signature/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+Changelog
+=========
+
+Unreleased
+----------
+
+1.0.0 - (August 31, 2017)
+------------------
+Initial stable release
```

Comment:
Please put this under the Unreleased section. A 1.0.0 section will be created with the release date when the component actually gets released.